neuralcoref/train/training_requirements.txt

```diff
@@ -1,3 +1,3 @@
 spacy
-torch
+torch>=0.4.0
```

Comment:
Looks good, thanks! I think it makes sense here to also specify an upperbound for the next future (potentially breaking) big release of torch (we should probably do the same for all dependencies)